### PR TITLE
remove adrianna from all files

### PR DESCRIPTION
**Pull request by iamNoah1**
Unfortunately, Adrianna stepped back from being a german reviewer :(

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Sad to hear. 😥😢

One point:
after this removal, there will be 2 `de` approvers (@iamNoah1 @DaveVentura),
so if you (@iamNoah1) open a PR, then @DaveVentura becomes the only one that can give an approval, and vice versa,
but we need at least 2 approvals for PRs to be merged.

Would it be needed to add @CathPag (or someone else) as an `de` approver? 😊
(As you may already know, if wee add @CathPag (or someone else) as an `de` approver,
then whenever a PR that updates files under `content/de/` is opened, GitHub will request review from her/him.)

**Comment by iamNoah1** [comment]
> Sad to hear. 😥😢
> 
> One point: after this removal, there will be 2 `de` approvers (@iamNoah1 @DaveVentura), so if you (@iamNoah1) open a PR, then @DaveVentura becomes the only one that can give an approval, and vice versa, but we need at least 2 approvals for PRs to be merged.
> 
> Would it be needed to add @CathPag (or someone else) as an `de` approver? 😊 (As you may already know, if wee add @CathPag (or someone else) as an `de` approver, then whenever a PR that updates files under `content/de/` is opened, GitHub will request review from her/him.)

@jihoon-seo yes I know. @CathPag can we add you again until we recruit a new reviewer?

**Comment by CathPag** [comment]
Sure. Happy to fill in

**Comment by iamNoah1** [comment]
Thanks @CathPag. @jihoon-seo so we need to assign Catherine explicitly as german reviewer again? Or can she review german content because she is maintainer?

File: CODEOWNERS
Comment (jihoon-seo) [inline CODEOWNERS:25]:
```suggestion
/content/de/ @iamNoah1 @DaveVentura @CathPag
```

**Review by jihoon-seo — COMMENTED** [review]
Yes right, even now she can review German content because she is maintainer.
The only difference of whether to add her to CODEOWNERS's `de` section explicitly would be
whether GitHub requests review from her explicitly or not (and accordingly send review request notification for `de` PRs to her explicitly or not). 😊
IMO adding her would be great, since she will get notified so will not miss. 😊

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Review by seokho-son — APPROVED** [review]
LGTM

**Pull request merged**
